package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
 		"commitplease": "2.0.0",
+		"globalize-compiler": "0.1.1",
```

Comment:
@rxaviers is this just very conservative versioning, or do you expect breaking changes coming up in the compiler?